src/common/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ set(efu_common_INC
   DumpFile.h
   EFUArgs.h
   EV42Serializer.h
+  Expect.h
```

Comment:
consider to move this to the file gccintel.h which was an attempt to collect compiler and architecture 
specific stuff. The idea was never carried through, but I think we should try to keep compiler 
dependent stuff together. Well at least have a look at gccintel.h :-)